### ISSUE TITLE
Use python -m towncrier

### ISSUE THREAD

**Issue by mauritsvanrees** [collective/zestreleaser.towncrier#17]
Instead of locating a `towncrier` executable, which may involve trickery and magic, we may be able to call `python -m towncrier`. So instead of this:

```
path = check_towncrier(data)
if not path:
    return
cmd = [path, '--version', data['new_version'], '--yes']
```

we could maybe do this:

```
cmd = [sys.executable, '-m', 'towncrier', '--version', data['new_version'], '--yes']
```

We could also make this one of the fallbacks in our `_towncrier_executable` function.

This might have helped with issue #6.

**Comment by mauritsvanrees** [comment]
I notice that with a buildout section like this, the `towncrier` script is not actually installed:

```
[releaser]
recipe = zc.recipe.egg
eggs =
    zest.releaser[recommended]
    zestreleaser.towncrier
```

Solution: either add `towncrier` to that list explicitly, or add an option to that section:

```
dependent-scripts = true
```

This is another reason why it would be helpful to call `python -m towncrier`: then it does not matter that the script is not installed.

**Comment by mauritsvanrees** [comment]
Fix released in 1.2.0.
